tests/test_call_parentheses.rs

```diff
@@ -0,0 +1,298 @@
+use stylua_lib::{format_code, CallParenType, Config, OutputVerification};
```

Comment:
Is this similar to `test_no_call_parens.rs`? Should we just delete the no_call_parens file and move any test cases over there into here, with `CallParenType::None`.

Reply:
It's pretty similar . I can move the tests . But having the no_call_parens tests also insures currently deprecated no_call_paren option doesn't get accidentally broken (I'm not sure if we care about this)

Reply:
Hm, true - leave it for now. When the option gets removed I can move over any new tests